contract/src/agoric-basics.contract.js

```diff
@@ -101,39 +143,45 @@ export const start = async zcf => {
   /** @type {OfferHandler} */
   const tradeHandler = buyerSeat => {
     // give and want are guaranteed by Zoe to match proposalShape
-    const { want } = buyerSeat.getProposal();
+    const { give, want } = buyerSeat.getProposal();
+
+    hasInventory(want.Tickets.value, inventory) ||
```

Comment:
Peeling off `.value` is something we avoid.

Consider minting the whole inventory at contract start to a seat held by the contract. You can then use `AmountMath.isGTE(inventorySeat.allocation('Tickets'), want.Tickets)`.

This assumes the contract is supposed to track inventory as it decreases and not just re-sell the same seats over and over. I should double-check that requirements doc...